renovate.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "schedule": ["before 9am on Monday"],
+  "timezone": "Europe/Paris",
+  "major": {
+    "ignoreDeps": ["pouchdb-find", "react-router", "cordova"]
+  }
```

Comment:
I see what you're trying to do, but I'm not 100% sure it will work because `ignoreDeps` is evaluated before `major` is calculated for a repo. Maybe one day I can get Renovate to support this concept, but not today.

I actually suggest you remove this entire `"major"` section out. Here's why:

- Renovate *will* suggest to you to upgrade major versions of these once you enable it, but you can simply close the PR unmerged. Renovate will not suggest that major version again (e.g. if `6.3.4` is out now and you close it, Renovate won't annoy you when `6.3.5` or `6.4.0` is released). Closing a major upgrade PR unmerged says to Renovate "I am not interested in this major upgrade. Do not suggest this major version again unless I have upgraded to it manually"
- If you *do* eventually upgrade these, then Renovate will automatically kick back into action with minor and patch updates, without you needing to edit your `renovate.json` to undo the ignore

Reply:
Wow ok, I didn't expect renovate to be smart like that. We should be perfectly fine without this section then 👍